### PR TITLE
clarify src:geom_alt and add def for src:alt_label

### PR DESCRIPTION
**Pull request by nvkelso**
Document the "new" property as proposed in https://github.com/whosonfirst-data/whosonfirst-data/issues/1714 and active since 2019.

### CODE REVIEW

File: properties/src/alt_label.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "id": 1913841421,
+    "name": "alt_label",
+    "prefix": "src",
+    "description": "The data source of a record's alternate geometry, in the alternate geometry file's property list. The record's default file property list should use src:geom_alt to list this specific alternate geometry source and any other alternates geometry sources. Valid property values are listed in the whosonfirst-sources repository.",
```
Comment (stepps00) [inline properties/src/alt_label.json:5]: Should we somehow include that this property is essentially the "source" + "extras" in a alt record's file name?

In most cases, an alt record's filename is constructed as: "id" + `alt` + "source" .geojson... but in some cases, a filename can be constructed as "id" + `alt` + "source" + "extras" .geojson

Ex: https://github.com/whosonfirst-data/whosonfirst-data-admin-us/blob/master/data/859/225/83/85922583-alt-mapzen-land.geojson?short_path=e3d1838

**Review by stepps00 — COMMENTED** [review]
Note about the alt_label property description.. otherwise, LGTM

File: properties/src/alt_label.json (hunk 1 above)
Comment (nvkelso) [reply properties/src/alt_label.json:5]: Good point. I think that's more relevant to `src:geom_alt` soI've added it there.

Comment (nvkelso) [reply properties/src/alt_label.json:5]: Via https://github.com/whosonfirst/whosonfirst-properties/pull/115/commits/acfb578402f72ec263e1206d42cbf3a1cd36b286

**Comment by nvkelso** [comment]
@stepps00 @thisisaaronland I don't see a Makefile target to regenerate all the README.md within each of the property dirs. What am I missing?

**Comment by nvkelso** [comment]
Ah OK. I've updated the README via https://github.com/whosonfirst/whosonfirst-properties/pull/115/commits/c506726ffdc7dcefb4da8edc6d85478159d4452f. 

Can I get a final review and ✅  @stepps00?

**Comment by stepps00** [comment]
LGTM!  ✅

**Pull request merged**